server/api.go

```diff
@@ -74,6 +85,10 @@ func NewAPI(p *Plugin, store store.Store) *API {
 	router.HandleFunc("/connected-users/download", api.getConnectedUsersFile).Methods(http.MethodGet)
 	router.HandleFunc(APIChoosePrimaryPlatform, api.choosePrimaryPlatform).Methods(http.MethodGet)
 
+	if os.Getenv("MM_MSTEAMSSYNC_MOCK_CLIENT") == "true" {
```

Comment:
Adding this new mock API based on environment variable, the implementation is bellow, and the idea is that you can decide what to mock using API calls.